.github/workflows/main.yml

```diff
@@ -40,9 +40,9 @@ jobs:
         env_vars: OS,PYTHON
         name: codecov-umbrella
         fail_ci_if_error: true
-    - name: Publish on PiPy
-      env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PIPY_USER_TOKEN }}
-      run: make publish-pipy
-    
+#    - name: Publish on PiPy
```

Comment:
why have you commented this?

Reply:
yes, this will be removed after the 1st release